tests/test_voltage_initializer.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2020-2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
```

Comment:
can you add 
SPDX-License-Identifier: MPL-2.0

Reply:
Done